### DISCUSSION TITLE
I found a very strange bug on GitHub

### DISCUSSION THREAD

**Discussion by oluqom** [community/community#31286] [Other Feature Feedback, Questions, & Ideas] [answered]
# Description of the bug

When I first load my account, there are years to the right of my activity and for some reason it is set to inline style, namely in the div where the list with years is located, the strange thing is that if you reload the page and even log out and log in again everything is fine, but if I I don’t log in for a long time, then it happens again, I tested it on all the browsers that I have (I’m a front-end developer) and therefore I have almost everything, and everything is the same there.

## This is what the bug looks like

![mfFSsQJ](https://i.imgur.com/mfFSsQJ.png)
You'd think I did it myself, but I don't have a split personality, although maybe...

**+ Does anyone else have one? It's just that my friends have the same crap.**

**Comment by melvincwng** [comment, marked as answer]
@1PopovAleksey Hi Aleksey, this is a known issue as seen from: https://github.com/community/community/discussions/30938

A member of the community has already raised a bug report to the GitHub Bug Report team, and they should be already looking into it.

Hope this answers your question 😊
